tests/lib/test_pkg.ml

```diff
@@ -1,6 +1,13 @@
 open Rresult
 open Dune_release
 
+(* compatibility shim with OCaml 4.06 and 4.07 *)
+module Option = struct
+  include Option
+
+  let map f = function None -> None | Some x -> Some (f x)
```

Comment:
We already have redefined `Option.map` in `Dune_release.Stdext`

Reply:
A bit unusual to include a labelled version there but I'll take it :-)